proto/raft.proto

```diff
@@ -65,7 +65,8 @@ message FileMetadata {
   FileRecord file_record = 1;
   StorageMetadata storage_metadata = 2;
 
-  // Size of the stored data (if data is compressed, this will be the compressed size).
+  // Size of the stored data (if data is compressed, this will be the compressed
+  // size).
   int64 size_bytes = 3;
```

Comment:
maybe we should rename this to stored_size_bytes?

Reply:
Done